Solutions/Marain.Workflow.Storage.Sql.Database/Tables/WorkflowInstanceInterest.sql

```diff
@@ -0,0 +1,7 @@
+﻿CREATE TABLE [dbo].[WorkflowInstanceInterest]
+(
+	[WorkflowInstance] INT NOT NULL , 
+    [Interest] NVARCHAR(50) NOT NULL, 
+    CONSTRAINT [PK_WorkflowInstanceInterest] PRIMARY KEY ([WorkflowInstance], [Interest]), 
```

Comment:
Does this automatically make this a clustered index? If not, should we? (Clustered indexes were, as I understood it, highly recommended in SQL Server. IIRC, early editions of SQL Azure didn't actually support creation of tables that didn't have a clustered index!)

Reply:
By default a primary key is a unique clustered index, I believe.

Reply:
Yes, it is `UNIQUE CLUSTERED` by default. You can opt in to making it `NONCLUSTERED`